charts/kestra/values.yaml

```
@@ -304,3 +304,5 @@ startupProbe:
   httpGetEnabled: true
   httpGetExtra: {}
 
+# -- Extra Kubernetes Manifests
+extraDeploy: []
```

```suggestion
extraManifests: []
```

Or `extraResources` as Kubernetes manage resources.